DemoApp/DemoApp/TMInputPage.xaml.cs

```diff
@@ -0,0 +1,29 @@
+using Trimble.Modus.Components;
+
+namespace DemoApp;
+
+public partial class TMInputPage : ContentPage
+{
+	public TMInputPage()
+	{
+		InitializeComponent();
+    }
+    private void modusInput_Focused(object sender, EventArgs e)
+    {
+        Console.WriteLine("Container focused");
+
+    }
+
+    private Tuple<bool, string> CustomValidateInput(object sender)
+    {
+        var input = sender as TMInput;
+        if(input.Text.Length > 3)
```

Comment:
This method can return null in which case the helper text should be displayed in the input.

Reply:
Done